go.mod

```diff
@@ -29,6 +29,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
```

Comment:
@AlgoStephenAkiki + other reviewers:  For my understanding - What's the plan to remove remaining dependencies so that go-algorand `go.mod` entry can be removed?

* I understand the PR description marks AVM assembler as out-of-scope.  However, I can imagine some solutions that change based on considering the assembler.
* I'm gauging if paths have been considered, if it's an open question or something else.

Reply:
Resolving - Thanks to discussion with @winder and @AlgoStephenAkiki, I learned the plan is:  In a forthcoming PR, remove the need for assembler usage by switching to state delta API.